README.md

```diff
@@ -120,41 +123,45 @@ Besides the environment variables supported by Vue CLI Wegue offers the followin
 
 ## Run with Docker
 
-Versioned Docker images are available on [DockerHub](https://hub.docker.com/r/meggsimum/wegue/tags).
-Run the `latest` (`master`) version of the Wegue Docker Image as follows:
+The shipped Dockerfile gives you a basic idea how to use Wegue with Docker.
+Maybe the Dockerfile needs some modification if you use custom application code.
+Once you get along with just modifying the application config JSON it should be sufficient to do the following steps:
+
+Build a Wegue Docker image as follows:
+
+``` bash
+docker build -t my-wegue-img:latest .
+```
+
+Start the freshly build image as container:
```

Comment:
```suggestion
Start the freshly build image as a container:
```

Reply:
Oops, correcting a detail and letting a real error here... Should have been Start the freshly **BUILT** image...